extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/AccessLogHandler.java

```diff
@@ -90,18 +93,26 @@
     private final AccessLogReceiver accessLogReceiver;
     private final String formatString;
     private final ExchangeAttribute tokens;
+    private final Pattern excludePattern;
 
-    public AccessLogHandler(final AccessLogReceiver accessLogReceiver, final String formatString, ClassLoader classLoader) {
+    public AccessLogHandler(final AccessLogReceiver accessLogReceiver, final String formatString, ClassLoader classLoader,
+            Optional<String> excludePattern) {
         this.accessLogReceiver = accessLogReceiver;
         this.formatString = handleCommonNames(formatString);
         this.tokens = new ExchangeAttributeParser(classLoader, Collections.singletonList(new SubstituteEmptyWrapper("-")))
                 .parse(this.formatString);
+        if (excludePattern.isPresent()) {
+            this.excludePattern = Pattern.compile(excludePattern.get());
+        } else {
+            this.excludePattern = null;
+        }
     }
 
     public AccessLogHandler(final AccessLogReceiver accessLogReceiver, String formatString, final ExchangeAttribute attribute) {
```

Comment:
`String formatString` Should it be final?  Like other constructor above.

Reply:
It makes no practical difference.